guides/plugins/plugins/content/media/extending-media-deletion-service.md

```diff
@@ -0,0 +1,135 @@
+# Extending the Media Deletion Service
```

Comment:
The header does not really describe the use case, but rather describes the technical things that are needed

Maybe it would make sense to rephrase along the lines of 
`Prevent deletion of media files referenced in your plugins`

What do you think?